src/Tribe/Rewrite.php

```diff
@@ -750,6 +760,8 @@ protected function get_dynamic_matchers( array $query_vars ) {
 	 * Returns a list of post types supported by the implementation.
 	 *
 	 * @since 4.9.11
+	 *
+	 * @return array<string> An array of post types supported and handled by the rewrite implementation.
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Function return type is not void, but function has no return statement
* :no_entry_sign: Line exceeds maximum limit of 100 characters; contains 104 characters

<sub>hash: dd889a9f204ae72bc0ab0b6d2f7327d6</sub>